templates/company_management/list_companies.html

```diff
@@ -0,0 +1,111 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+
+    <div class="row">
+        <div class="col-sm-12">
+            <div class="element-wrapper">
+                <h6 class="element-header">
+                    Companies List
+                </h6>
+                <div class="element-box-tp">
+                    <!----------------------------
```

Comment:
i think we do not need the above controls.

Reply:
done